onnx/helper.py

```diff
@@ -343,20 +370,32 @@ def printable_value_info(v):
     return s
 
 
-def printable_node(node, prefix=''):
+def printable_node(node, prefix='', subgraphs=False):
```

Comment:
Is this just for backward compatibility or is there real cases that we don't want to print the subgraphs?

Reply:
Do we even have any case which contain subgraph as attribute?

Reply:
I basically just wanted to leave it up to the caller to handle the subgraphs. Notice that if the flag is true there's an extra return value and the caller can do whatever they want with it

Reply:
I would say let's always print the subgraphs, and do not have different number of return values. Returning graph objects in a print function doesn't sound appropriate to me.

Reply:
I think that makes sense in isolation, but in context this is used in a recursive descent algorithm. Printing the graph within the op doesn't look good, so what's happening now is we propagate all graphs up to the top-level (print_graph) and print them one after the other. This makes it look like a normal SSA organized into basic blocks